extra/modules/greenbids-real-time-data/src/main/java/org/prebid/server/hooks/modules/greenbids/real/time/data/model/result/GreenbidsInvocationResult.java

```diff
@@ -0,0 +1,108 @@
+package org.prebid.server.hooks.modules.greenbids.real.time.data.model.result;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Imp;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.core.Partner;
+import org.prebid.server.hooks.v1.InvocationAction;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+public class GreenbidsInvocationResult {
+
+    private static final int RANGE_16_BIT_INTEGER_DIVISION_BASIS = 0x10000;
+
+    public BidRequest updatedBidRequest;
+
+    public InvocationAction invocationAction;
+
+    public AnalyticsResult analyticsResult;
```

Comment:
why public?

Reply:
use `@Value` and `@Builder` lombok annotation if you want to create an immutable object (and usually you want)

Reply:
fixed this in `GreenbidsInvocationResult` and `GreenbidsInferenceData`